tip-calculator/index.js

```diff
@@ -0,0 +1,14 @@
+function hackTip() {
+    var bill = document.getElementById("bill").value;
```

Comment:
have you considered using 'let' instead of 'var'?

Reply:
Yes, it is typically what I use. This is my first hackathon, and the super quick turnaround just threw me for a loop to start. This is something I will definitely refactor to fit with my standard code style.